transit_odp/timetables/views/review.py

```diff
@@ -203,6 +263,27 @@ def get_context_data(self, **kwargs):
                 context["booking_arrangements"] = booking_arrangements_info[0][0]
                 context["booking_methods"] = booking_arrangements_info[0][1:]
 
+        date = self.request.GET.get("date", datetime.now().strftime("%Y-%m-%d"))
+        target_date = datetime.strptime(date, "%Y-%m-%d").date()
+        timetable_inbound_outbound = TimetableVisualiser(
+            revision_id, service_code, line, target_date
+        ).get_timetable_visualiser()
+
+        # Set the context for the timetable visualiser and the line details
+        context["curr_date"] = date
+        for direction in ["outbound", "inbound"]:
+            direction_details = timetable_inbound_outbound[direction]
+            journey = direction_details["description"]
+            journey = direction.capitalize() + " - " + journey if journey else ""
+            bound_details = self.get_direction_timetable(
+                direction_details["df_timetable"], "outbound"
+            )
+            context[direction + "_timetable"] = bound_details["df_timetable"]
+            context[direction + "_total_page"] = bound_details["total_page"]
```

Comment:
When uploading file `BODS_PB0001746_15_20230108_21 1.xml` the third page doesnt have any journey codes displayed

![image](https://github.com/department-for-transport-BODS/bods/assets/95031756/6e3aa1d0-a5cf-4b7d-9386-92a9a65663b9)

Reply:
fixed and pushed